src/interfaces/slack_bot/.env.example

```diff
@@ -0,0 +1,23 @@
+# Environment variables declared in this file are automatically made available to Prisma, and to the application when NODE_ENV=development.
+# See README.md and https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+TOOLKIT_API_HOST=http://0.0.0.0:8000
+COHERE_ORGANIZATION_ID=<cohere-organization-id>
+COHERE_SLACK_TEAM_ID=<cohere-slack-team-id>
+SLACK_SIGNING_SECRET=<signing-secret>
+FEEDBACK_CLIENT_TAG=<feedback-client-tag>
+COMMAND_SUFFIX=<app-name>
+SENTRY_DSN=<sentry-dsn>
+DATABASE_URL="postgresql://coralslack:password@localhost:5432/coralslackdb"
```

Comment:
Is this pointing to the same DB that's run with our backend docker service?

If so, should it have the same user:pass by default as that one

Reply:
It does not use the same database as the Toolkit docker service. At the present time the user is required to set up a PostgreSQL database on their own, and configure the URL here. In the future, it would make sense to Dockerize the Slack bot interface, but I think that is out of scope for the initial slack bot PR.